span-processing-config-service-api/src/main/proto/org/hypertrace/span/processing/config/service/v1/span_processing_config_service.proto

```diff
@@ -149,3 +150,46 @@ message ApiNamingRuleMetadata {
   google.protobuf.Timestamp creation_timestamp = 1;
   google.protobuf.Timestamp last_updated_timestamp = 2;
 }
+
+message SamplingConfig {
+  string id = 1;
+  Scope scope = 2;
+  RateLimitConfig rate_limit_config = 3;
+  SpanFilter filter = 4;
+  bool disabled = 5;
+}
+
```

Comment:
The idea is to send the inclusion rules as disabled sampling config(with negated filters) with the highest priority and API level override scoping.

Reply:
The reasoning behind introducing disabled field is the following - If we are to send the inclusion rules modified as sampling config, we have to specify a way to say that the sampling rate(I mean per traces, not per time frame) must be 100%. Since we do not have this concept of rate(traces to sample/total) here(we specify traces to sample/time frame), added `disabled`.
`disabled` would be `false` for all the sampling configs except for the inclusion rules related ones, for which it would always be `true`.